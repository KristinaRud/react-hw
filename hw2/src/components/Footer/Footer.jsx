import { Component } from "react";
import "./Footer.scss";

export default class Footer extends Component {
  render() {
    return (
      <footer className="g-footer">
        <div className="copywriting">Footer (c) 2023</div>
      </footer>
    );
  }
}
