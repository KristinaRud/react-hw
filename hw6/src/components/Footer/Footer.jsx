import "./Footer.scss";

const Footer =()=>{
  return (
    <footer className="g-footer">
      <div className="copywriting">Footer (c) 2023</div>
    </footer>
  );
}

export default Footer;
