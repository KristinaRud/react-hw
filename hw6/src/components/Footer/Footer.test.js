import Footer from "./Footer";
import {render} from "@testing-library/react";

describe('Footer snapshot testing', ()=>{
    test('should Footer match snapshot', () => {
        const {asFragment} = render(<Footer/>);
        expect(asFragment()).toMatchSnapshot();
    })
})