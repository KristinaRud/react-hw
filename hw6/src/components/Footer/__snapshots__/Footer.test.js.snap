// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot testing should Footer match snapshot 1`] = `
<DocumentFragment>
  <footer
    class="g-footer"
  >
    <div
      class="copywriting"
    >
      Footer (c) 2023
    </div>
  </footer>
</DocumentFragment>
`;
